src/components/Loader/Loader.jsx

```diff
@@ -0,0 +1,92 @@
+import { useEffect, useState } from "react";
+import styles from "./Loader.module.scss";
+import { ReactComponent as First } from "./svg/svg1.svg";
+import { ReactComponent as Second } from "./svg/svg2.svg";
+import { ReactComponent as Third } from "./svg/svg3.svg";
+import { ReactComponent as Four } from "./svg/svg4.svg";
+import { ReactComponent as Five } from "./svg/svg5.svg";
+
+export const Loader = () => {
+  const [animationStates, setAnimationStates] = useState({
+    second: false,
+    third: false,
+    four: false,
+    five: false,
+  });
```

Comment:
move away from component this state as a `ANIMATION_DEFAULT_VALUE`

it will looks like:
```
const ANIMATION_DEFAULT_VALUE = {
seond: false,
third: false,
four: false,
five: false,
}

const Loder = () => {...}
```